pkg/operator/vars.go

```diff
@@ -0,0 +1,75 @@
+//
+// Copyright (c) 2012-2018 Red Hat, Inc.
+// This program and the accompanying materials are made
+// available under the terms of the Eclipse Public License 2.0
+// which is available at https://www.eclipse.org/legal/epl-2.0/
+//
+// SPDX-License-Identifier: EPL-2.0
+//
+// Contributors:
+//   Red Hat, Inc. - initial API and implementation
+//
+package operator
+
+import (
+	"github.com/eclipse/che-operator/pkg/util"
+	"strings"
+)
+
+var (
+	// general config
+	cheFlavor           = util.GetEnv("CHE_FLAVOR", "che")
+	namespace           = util.GetNamespace()
+	protocol            = "http"
```

Comment:
Can you elaborate why it is hardcoded?

Reply:
Che flavor is a default that can be overridden in a configmap

Reply:
I mean protocol